chore: generated CHANGELOG for origin/release/v52.5.0 (#3415)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

# 52.5.0 - July 14, 2021

## Added

#### salesforcedx-vscode-apex-replay-debugger

> - Update test sidebar icon coloring with test results and add test coverage ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))

Reviewer: ```suggestion
- Update test sidebar icons and add test coverage for easy button replay debugger ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))
```

#### salesforcedx-vscode-core

> - Conflict detection update modal ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))

Reviewer: ```suggestion
- Update modal text for conflict detection ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
```

> - Conflict detection update modal ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))

Reviewer 2: ```suggestion
- Update text on Conflict detection modal window ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
```

> - Conflict detection UI enhancement ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))

Reviewer: ```suggestion
- Enhance hovertip text for conflict detection ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
```

> - Conflict detection UI enhancement ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))

Reviewer 2: ```suggestion
- Conflict detection UI enhancement to show last modified date information ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
```

## Fixed

#### docs

> - Doc Update for Conflict Detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))

Reviewer: ```suggestion
- Update docs for conflict detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
```

> - Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))

Reviewer: ```suggestion
- Disable conflict detection on `SFDX: Retrieve **` commands ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```

#### salesforcedx-vscode-core

> - Remove setting to enable function @W-9311863 ([PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))

Reviewer: ```suggestion
- Remove setting to enable function [PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))
```

> - Remove setting to enable function @W-9311863 ([PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))

Reviewer 2: You can get rid of the GUS workitem Id

> - Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))

Reviewer: Incorporate same change as above

> - Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))

Reviewer 2: We can get rid of this. We cherry picked this change to last week's release.

Author: Sounds good, removed

> - Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))

Author: ```suggestion
- Disable conflict detection on `SFDX: Retrieve **` commands ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```
